tests/report/common/tests.cpp

```diff
@@ -301,144 +328,294 @@ TEST_FCN void TestLocalReport(void* args_)
      *     3. opt_params can be zeroed out target info.
      *     4. opt_params can be a valid target info.
      */
-    {
-        report_size = 1024 * 1024;
-        OE_TEST(
-            GetReport(
-                0,
-                NULL,
-                0,
-                NULL,
-                sizeof(opt_params),
-                report_buffer,
-                &report_size) == OE_INVALID_PARAMETER);
-        OE_TEST(
-            GetReport(0, NULL, 0, opt_params, 5, report_buffer, &report_size) ==
-            OE_INVALID_PARAMETER);
+    {{report_size = 1024 * 1024;
```

Comment:
Formatting looks weird here

Reply:
um, yeah! But that is what the formatter did. I got rid of the extra block and it seemed to make it better.